### ISSUE TITLE
Can't force export to .mp4? 

### ISSUE THREAD

**Issue by CuddleBear92** [k4yt3x/video2x#113]
Have some .flv video streams i extracted from flash files i didn't want to encode into a video format cause of the extra time, storage and quality loss it might entail. 

video2x exports the final file into an .flv file again when h265 isn't supported, this makes it error out.

Tried adding `"-f": "mp4"` to the video2x.json under `"migrating_tracks":`

That sadly didn't work. HEVC is obviously not compatible with .flv at all and i rather have it in .mp4 anyway.

An option for output extension or a workaround for this maybe? 

TLDR; want to be able to input .flv video streams and output .mp4 files.

**Comment by k4yt3x** [comment]
It should output H.264 by default.
See the following code section for output options. Modify anything here if you need to.
This section should be modified as well on top of migrating_tracks.

https://github.com/k4yt3x/video2x/blob/8b845e35b37c8bb999a1ddb3732a04ea28cb5539/bin/video2x.json#L65-L70

**Comment by CuddleBear92** [comment]
But setting it back to libx264 wouldnt make a difference cause .flv doesn't support that anyway. 
also would require me to encode it at a much higher cq and size to get the same quality when i am suppose to run it though an svp encode after the fact. hmmm
Yeah, the issue is that it packages it down to an .flv instead of an .mp4 or .mkv.

The encoding process and frame extraction itself works great. just the final step in the encoding process which is packaging the container that is the issue here.

**Comment by k4yt3x** [comment]
@CuddleBear92 what's the command that you're using?
I believe specifying `-o output.[extension]` should work.

**Comment by CuddleBear92** [comment]
I am just doing like: 
```
video2x -i "D:\Programs\Video2X\Input" -o "D:\Programs\Video2X\Output" -m cudnn -d waifu2x_caffe -y "D:\Programs\Waifu2x-caffe\models\photo" -r 1.5
pause
```

So just simple input-output folders.
Unsure how to set the extension while keeping the rest of the filenames. 
Errors in the filenames like `originalname.flv.mp4` is fine too i guess

**Comment by k4yt3x** [comment]
With folders, video2x just copies the input codec.
